include/zephyr/bluetooth/services/hrs.h

```diff
@@ -19,10 +19,49 @@
 
 #include <stdint.h>
 
+#include <zephyr/sys/slist.h>
+
 #ifdef __cplusplus
 extern "C" {
 #endif
 
+/** @brief Heart rate service callback structure */
+struct bt_hrs_cb {
+	/** @brief Heart rate notifications changed
+	 *
+	 * @param enabled Flag that is true if notifications were enabled, false
+	 *                if they were disabled.
+	 */
+	void (*ntf_changed)(bool enabled);
+
+	/** Internal member to form a list of callbacks */
+	sys_snode_t node;
+};
+
+/** @brief Heart rate service callback register
+ *
+ * This function will register callbacks that will be called in
+ * certain events related to Heart rate service.
+ *
+ * @param cb Pointer to callbacks structure
```

Comment:
We are taking a mutable borrow of the `struct bt_hrs_cb`. Let's document that. Suggestion follows:

```suggestion
 * @param cb Pointer to callbacks structure. The target object is given to the stack for exclusive use and the memory must remain valid until unregistered.
```

Reply:
Added missing information.